src/app/components/Figure/Caption/index.stories.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
 import Caption from './index';
+import InlineLink from '../../InlineLink';
 
-storiesOf('Caption', module).add('default', () => (
-  <Caption>This is a caption.</Caption>
-));
+storiesOf('Caption', module)
+  .add('default', () => <Caption>This is a caption.</Caption>)
+  .add('containing an inline link', () => (
+    <Caption>
+      This is a caption{' '}
```

Comment:
What's `{' '}` achieving here?

Reply:
Good question!

Reply:
This is what prettier/eslint config does to keep a space after the work `caption` and before the `<InlineLink>` element. You can work around it by saving the text in a variable. e.g. `const captionText = 'This is a caption '` & use it in the element.

Reply:
```suggestion
      {'This is a caption '}
```